core/mpi/get_info.cmake

```
@@ -0,0 +1,17 @@
+ginkgo_print_module_header(${detailed_log} "MPI")
+ginkgo_print_variable(${detailed_log} "GINKGO_FORCE_CUDA_AWARE_MPI")
+ginkgo_print_variable(${detailed_log} "GKO_CUDA_AWARE_RUN_STATUS")
+ginkgo_print_variable(${detailed_log} "MPI_C_COMPILER")
+ginkgo_print_variable(${detailed_log} "MPI_CXX_COMPILER")
+ginkgo_print_variable(${detailed_log} "MPI_CXX_COMPILE_OPTIONS")
+ginkgo_print_variable(${detailed_log} "MPI_CXX_LINK_FLAGS")
+ginkgo_print_variable(${detailed_log} "MPI_CXX_LIB_NAMES")
+ginkgo_print_variable(${detailed_log} "MPI_CXX_HEADER_DIR")
+ginkgo_print_variable(${detailed_log} "MPI_mpi_LIBRARY")
```

Where is this variable specified? Did you mean 
```suggestion
ginkgo_print_variable(${detailed_log} "MPI_CXX_LIBRARY")
```